src/Components/Taha/Index.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import "./style.css";
+
+function Index() {
+  return (
+    <section className="taha">
```

Comment:
As a shortcut and to reduce the implementation number of lines we can replace this 

```
 const Index = () => {
  return()
}
```

with this
` const Index = () => () `